tools/cldr-code/src/main/java/org/unicode/cldr/test/CheckExemplars.java

```diff
@@ -302,6 +302,16 @@ public CheckCLDR handleCheck(
 
     private void checkParse(
             String path, String fullPath, String value, Options options, List<CheckStatus> result) {
+        if (value == null) {
```

Comment:
It is very strange that a null value could occur here. Probably only via a manually modified XML file, since it would have to be a path that didn't occur in root.

Reply:
it's from the unit test. so, it's just a robustness issue, which is presumably what the test is trying to check?